invesalius/data/viewer_volume.py

```diff
@@ -926,8 +926,27 @@ def ActivateTargetMode(self, evt=None, target_mode=None):
                 if self.obj_projection_arrow_actor:
                     self.obj_projection_arrow_actor.SetVisibility(1)
 
-    def OnUpdateObjectTargetGuide(self, m_img, coord):
+    def ComputeRelativeDistanceToTarget(self, m_img):
```

Comment:
I would move this method to other related module such as `coregistration`. We should leave the `viewer_volume `only doing viewer operations.

Reply:
Done